newIDE/app/src/UI/SemiControlledAutoComplete.js

```diff
@@ -276,6 +277,12 @@ export default React.forwardRef<Props, SemiControlledAutoCompleteInterface>(
                       props.onChange(event.currentTarget.value);
                       if (props.onBlur) props.onBlur(event);
                     },
+                    onMouseDown: (
+                      event: SyntheticMouseEvent<HTMLInputElement>
+                    ): void => {
+                      if (input.current && !input.current.value.length)
+                        setIsMenuOpen(!isMenuOpen);
+                    },
```

Comment:
I've added this to toggle menu with a click when input is empty. The menu used to stay open even when the user clicks to close it.

Reply:
Thanks for adding this 👍